tonic-build/src/prost.rs

```diff
@@ -610,8 +610,8 @@ impl Builder {
         self.compile_protos(protos, includes)
     }
 
-    /// Compile the .proto files and execute code generation using a
-    /// custom `prost_build::Config`.
+    /// Compile the .proto files and execute code generation using a custom
+    /// `prost_build::Config`. The provided config will be updated by this builder's config.
```

Comment:
Nit: suggest replacing "by" with "with" (also for the other method).

Reply:
Applied the suggestion.